charts/osm/values.yaml

```diff
@@ -14,11 +14,13 @@ OpenServiceMesh:
     retention:
       time: 15d
   certficateManager: tresor
+  enableVault: false
   vault:
     host:
     protocol: http
     token:
     role: openservicemesh
+  enableCertManager: false
```

Comment:
I don't see these 2 options being exposed via the install cli command, without which one can't override this using the `osm install` command. Refer to https://github.com/openservicemesh/osm/blob/main/cmd/cli/install.go#L192 on how these chart values are updated by the cli.